src/main/webapp/js/adminHome.js

```diff
@@ -0,0 +1,119 @@
+function createRowForResultTable(shortName, name, email, institution, isSuccess, status) {
+    var result = "<td>" + shortName + "</td>";
+    result += "<td>" + name + "</td>";
+    result += "<td>" + email + "</td>";
+    result += "<td>" + institution + "</td>";
+    if (isSuccess) {
+        result += "<td>Success</td>";
+    } else {
+        result += "<td>Fail</td>";
+    }
+    result += "<td>" + status + "</td>";
+    if (isSuccess) {
+        result = "<tr class=\"success\">" + result + "</tr>";
+    } else {
+        result = "<tr class=\"danger\">" + result + "</tr>";
+    }
+    return result;
+}
+
+var paramsCounter = 0;
+var paramsList = [];    // list of parameter strings that will be sent via ajax
+var instructorDetailsList = [];
+var isInputFromFirstPanel = false;
+
+function disableAddInstructorForm() {
+    $(".addInstructorBtn").each(function() {
+        $(this).html("<img src='/images/ajax-loader.gif'/>");
+    });
+    $(".addInstructorFormControl").each(function() {
+        $(this).prop('disabled', true);
+    });
+    
+}
+
+function enableAddInstructorForm() {
+    $(".addInstructorBtn").each(function() {
+        $(this).html("Add Instructor");
+    });
+    $(".addInstructorFormControl").each(function() {
+        $(this).prop('disabled', false);
+    });
+}
+
+function addInstructorByAjaxRecursively() {
+    $.ajax({
+        type : 'POST',
+        url :   "/admin/adminInstructorAccountAdd?" + paramsList[paramsCounter],
+        beforeSend : function() {
+            disableAddInstructorForm();
+        },
+        error : function() {
+            var rowText = createRowForResultTable("", "", "", "", false, "Cannot send Ajax Request!");
+            $("#addInstructorResultTable tbody").append(rowText);
+            if (isInputFromFirstPanel) {
+                var errorInstructorsText = $("#addInstructorDetailsSingleLine").val() + instructorDetailsList[paramsCounter] + "\n";
```

Comment:
just a suggestion, we can rename this to something like: instructorsToBeRetried
the current name is just as fine